scripts/docker/borgserver

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -eux
+if [ -n "${SSH_KEY:-}" ]; then
+  dpkg-reconfigure openssh-server
+  sed -i \
+    -e 's/^#PasswordAuthentication yes$/PasswordAuthentication no/g' \
+    -e 's/^PermitRootLogin without-password$/PermitRootLogin no/g' \
+    /etc/ssh/sshd_config
+  if [[ $(ls /etc/ssh-keys/ssh_host_*) ]]; then
```

Comment:
AFAIK one should not use [ls in scripts/parse it](https://unix.stackexchange.com/a/129120/146739). Okay, maybe there is not much parsing happening, but I you may anyway get crazy edge-cases/it is just not clean when you e.g. have spaces or so in there…
What I wanna say: Maybe there is a better solution? :slightly_smiling_face: 

Also, were these scripts tested with [shellcheck](https://www.shellcheck.net/)? (Just pasted this script and it did not complain to my surprise, so yeah,hmm…)